internal/core/services/workers_manager/workers_manager.go

```diff
@@ -196,3 +203,16 @@ func (w *WorkersManager) getDispensableWorkers(schedulers []*entities.Scheduler)
 	return dispensableWorkers
 
 }
+
+func getRunningComponent(builder workers.WorkerBuilder) string {
+	switch reflect.ValueOf(builder).Pointer() {
+	case reflect.ValueOf(operation_execution_worker.NewOperationExecutionWorker).Pointer():
+		return "operation_execution_worker"
+	case reflect.ValueOf(runtime_watcher_worker.NewRuntimeWatcherWorker).Pointer():
+		return "runtime_watcher_worker"
+	case reflect.ValueOf(metricsreporter.NewMetricsReporterWorker).Pointer():
+		return "metrics_reporter_worker"
+	default:
+		return "unknown"
```

Comment:
Hey @guilhermocc reflection is not a good practice to use on Golang (including [decreasing the performance](https://betterprogramming.pub/understand-reflect-in-go-24a68fcf1011)), what do you think instead of defining this `getRunningComponent` creating a method like what we do on `operations` called `Name()` or something like that and use this to infer which worker we are getting metrics?